### PR TITLE
[bitnami/keycloak] Revisit Goss/Cypress tests

### PR DESCRIPTION
**Pull request by FraPazGal**
### Description of the change

After reviewing the existing tests and the general testing strategy we have been following until now, we saw there was a need to revisit them. The main objective that our chart testing strategy should aim for is asserting that the Chart deliverable works. As such, the focus should not be on the application itself or the container, which are regarded as trustful components (that is, they should have been tested at a previous stage).

As charts usually pack a number of different components, it is essential to test that their integrations and the application itself work as expected.

Consequently, we have updated the tests so they are related to the chart deliverable and left only a couple of basic tests to verify that the application functions correctly. Besides that, an effort has been made to reduce future tests' flakiness improving some existing selectors and checks.

Tested in https://github.com/FraPazGal/charts/pull/73

> The VIB Verify executions failed due to the trivy action (now removed from `bitnami/charts` no longer present in `bitnami/charts`

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/keycloak/cypress/cypress/support/commands.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const COMMAND_DELAY = 1100;
+const COMMAND_DELAY = 500;
```
Comment (agomezmoron) [inline .vib/keycloak/cypress/cypress/support/commands.js:1]: Not blocker for now but we should unify all the COMMAND_DELAY to ensure all tests are aligned. Waiting 500ms or 1300ms (the maximum value I observed in the repo) is nothing after reducing the number of tests. Tuning too much the delays to reduce them at the maximum, sometimes, can give us false positives (due to extra needed delay). IMHO, it's something non blocker now but should be unified in another PR.

**Review by agomezmoron — APPROVED** [review]
LGTM but please, review the comment I left for a future PR unifying all (but please, do not block this).

File: .vib/keycloak/cypress/cypress/support/commands.js (hunk 1 above)
Comment (FraPazGal) [reply .vib/keycloak/cypress/cypress/support/commands.js]: Good idea, I created https://github.com/bitnami/charts/pull/11617 and updated this PR accordingly.

**Review by agomezmoron — APPROVED** [review]
LGTM, thanks for applying the change

**Review by joancafom — APPROVED** [review]

**Pull request merged**